packages/client/src/App.sass

```diff
@@ -246,6 +250,10 @@ h2
 .button.is-small.with-icon>svg
   transform: scale(0.7)
 
+/********************* Custom utility css - Columns ********************/
+.column
+  padding-left: 0
```

Comment:
yeah we should probably just add `pl-0` wherever we dont want the default column padding since a lot of components use this class and could have unintended side effects

Reply:
You're right, reverting this!